### ISSUE TITLE
[FEATURE REQUEST] FIle placeholder while uploading

### ISSUE THREAD

**Issue by parneet-guraya** [owncloud/android#4272]
We can add a  placeholder for a file which is being uploaded and show in the list where it is being added. This would be a nice improvement and would be a better acknowledgement to the user.

That's how GDrive does it -->

<img src='https://github.com/owncloud/android/assets/111801812/1c837058-be64-4ac5-a821-f60ee0db90f2' width='300'>

**Comment by JuancaG05** [comment]
This could be nice, but it's also a potential source of bugs since there are a lot of possible corner cases (and not so corner). I think it could be a bit complex technically as well.
At the moment, the latest improvement for this was the auto-refresh and it's so recent (https://github.com/owncloud/android/issues/4103), I would say this can wait, but let's keep it in the backlog.

**Comment by parneet-guraya** [comment]
Yes, I can understand that this can be a bit complex and lead to subtle bugs and yes let's keep it in backlog and I would surely like to work on it whenver possible :smile:
